2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,34 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+  while (list.firstChild) {
+    list.removeChild(list.firstChild);
+  }
+
+  todos.forEach((item) => {
+    let liEl = document.createElement("li");
+    liEl.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+    liEl.innerText = item.task;
+
+    let spanEl = document.createElement("span");
+    spanEl.className = "badge bg-primary rounded-pill";
+
+    let iEl1 = document.createElement("i");
+    iEl1.className = "fa fa-check";
+    iEl1["aria-hidden"] = "true";
+
+    let iEl2 = document.createElement("i");
+    iEl2.className = "fa fa-trash" ;
+    iEl2["aria-hidden"] = "true";
+
+    spanEl.appendChild(iEl1);
+    spanEl.appendChild(iEl2);
+    liEl.appendChild(spanEl);
+    list.appendChild(liEl);
+  });
+
```

Comment:
you look to be missing a bit of functionality for your icons - for example on click on the delete icon the item should be removed from the list & on click of the check icon it should be crossed out - let me know if you want a hand going over these :)